views/new_folder.templ

```diff
@@ -0,0 +1,65 @@
+package views
+
+import (
+	"github.com/ugent-library/deliver/ctx"
+	"github.com/ugent-library/deliver/models"
+	"github.com/ugent-library/okay"
+)
+
+type NewFolderArgs struct {
+	Folder *models.Folder
+	Errors *okay.Errors
+}
```

Comment:
Decided to put the args struct here and use it from `spaces.go`. Not sure if this is the right place to put it.

Reply:
would also have put it here